js/recipe/metadata.js

```diff
@@ -0,0 +1,27 @@
+import { recipeParentElement } from "../constants.js"
+import { render } from "../htmlRenderer.js"
+import * as els from "../htmlRenderer.js"
+
+export function metadata({ tags, name, datePublished, recipeYield }) {
+  render(recipeParentElement, els.h1({className: "post-title", innerHTML: name}));
+
+  const date = new Date(datePublished).toDateString();
+  const tagElements = tags.map(tag => els.span({innerHTML: `${tag}&nbsp`}));
+
+  const metaChildren = [
+    els.time({innerHTML: date}),
+    els.span({
+      className: "tags",
+      innerHTML: "/ Tags: &nbsp",
+      children: [
+        els.span({children: tagElements})
+      ]
+    }),
```

Comment:
I cannot get this tags section to display properly, I would like to pair with you on it and figure out what I'm missing!

Reply:
Solution: I was using innerHTML _and_ children which is not allowed in our implementation.